cibuildwheel/frontend.py

```diff
@@ -42,7 +42,9 @@ def _get_verbosity_flags(level: int, frontend: BuildFrontendName) -> list[str]:
             return ["-" + level * "v"]
         if level < 0:
             return ["-" + -level * "q"]
-    elif not 0 <= level < 2:
+    elif level > 1:
+        return ["-v"]
+    elif level < 0:
         msg = f"build_verbosity {level} is not supported for {frontend} frontend. Ignoring."
         log.warning(msg)
     return []
```

Comment:
I think there are a few changes required here to match the table in readme. e.g. -1 on pip doesn't do anything. It might be worth changing this to a map, with only a few supported settings per frontend, warning otherwise; up to you.